app/views/shared/views/_show_containers_dashboard.html.haml

```diff
@@ -0,0 +1,125 @@
+.container-fluid.container-tiles-pf.containers-dashboard.miq-dashboard-view{"ng-app"        => "containerDashboard",
+                                                                            "ng-controller" => "containerDashboardController as dashboard"}
+  .row.row-tile-pf
+    .col-xs-6.col-sm-4.col-md-2
+      .col-xs-12.provider-card{"pf-card"         => "",
+                               "show-top-border" => "true",
+                               "ng-if"           => "providerTypeIconClass"}
```

Comment:
Any chance we could make the if a bit more specific? This will pretend to be the other thing whenever `iconClassForProvider` encounters unknown provider type...

But maybe that's for a separate PR because looking at the code, `iconClassForProvider` seems to use a global variable, and the `$location` based logic in `refresh()` probably warrants a separate function .. I'm thinking let's make it *really* explicit and just make refresh call that funciton and set `$scope.isSingleProvider` (or something like that, not quite sure about the right name now.)